apps/wpcom-block-editor/src/default/features/rich-text.js

```diff
@@ -1,24 +1,52 @@
 /* global wpcomGutenberg */
 import { RichTextToolbarButton } from '@wordpress/block-editor';
+import { getBlockType } from '@wordpress/blocks';
 import { compose, ifCondition } from '@wordpress/compose';
 import { withSelect, withDispatch } from '@wordpress/data';
 import { registerFormatType } from '@wordpress/rich-text';
-import { get } from 'lodash';
 
-const RichTextJustifyButton = ( { blockId, isBlockJustified, updateBlockAttributes } ) => {
-	const onToggle = () =>
-		updateBlockAttributes( blockId, { align: isBlockJustified ? null : 'justify' } );
+const RichTextJustifyButton = ( {
+	blockId,
+	deprecatedIsBlockJustified,
+	styleAttributes,
+	updateBlockAttributes,
+} ) => {
+	const isBlockJustified = 'justify' === styleAttributes.typography?.textAlign;
+
+	const onToggle = () => {
+		// TODO: Remove this once we know all Atomic sites are on Gutenberg 22.1 or higher
```

Comment:
@mikachan, how can we make sure we'll get back to this? Is there a Linear project specifically for Atomic Gutenberg upgrades?

Reply:
GB 22.1 will be part of WordPress 7.0 but not WP6.9. Given that WoA admins are allowed to deactivate the GB plugin, I assume we'll have to keep this until everyone is on WP7.0, next spring.

(though arguably if they deactivate the plugin it's their own problem?)